ipaclient/install/clintpconf.py

```diff
@@ -0,0 +1,118 @@
+import os
+from logging import getLogger
+
+from ipaplatform.tasks import tasks
+from ipaplatform.paths import paths
+from ipapython import ipautil
+from ipapython import ntpmethods
+from ipapython.ntpmethods import detect_time_server
+
+logger = getLogger(__name__)
+
+
+class BaseNTPClient(object):
+    def __init__(self, fstore=None, ntp_confile=None, ntp_bin=None,
+                 statestore=None, cli_domain=None, timeout=None,
+                 flag=None, ntp_servers=None, ntp_pool=None,
+                 pre_args=None, post_args=None):
+
+        self.fstore = fstore
+        self.ntp_confile = ntp_confile
+        self.ntp_bin = ntp_bin
+        self.statestore = statestore
+        self.cli_domain = cli_domain
+        self.ntp_pool = ntp_pool
+        self.ntp_servers = ntp_servers
+        self.timeout = str(timeout)
+        self.flag = flag
+        self.pre_args = pre_args
+        self.post_args = post_args
+
+        if not pre_args and not post_args:
+            self.args = [paths.BIN_TIMEOUT, self.timeout,
+                         self.ntp_bin, self.flag, self.ntp_confile]
```

Comment:
AFAIK when you edit the default configuration file for chrony you do not need to start daemon with -q option. Only step necessary is to start the service and it loads changed configuration file at least for chrony. I believe that ntpd support this approach as well but i am not sure about openntpd

Reply:
Ok, check it out.

Reply:
I check :)

Reply:
Thanks, I will try to provide more feedback while you investigate this.

Reply:
Yes, when using the standard configuration file, it is enough to start the service. This is suitable for openntpd too. I already fixed.